fdbserver/storageserver.actor.cpp

```diff
@@ -873,6 +872,25 @@ ACTOR Future<Version> waitForVersion( StorageServer* data, Version version ) {
 		}
 	}
 }
+ 
+Future<Version> waitForVersion(StorageServer* data, Version version) {
+	try {
+		if (version == latestVersion) version = std::max(Version(1), data->version.get());
```

Comment:
Being picky but since you are touching this part of the code, it might be beneficial to unify the convention here: mixed usage of single line `if` statement; separate line but with no curly braces and etc..